src/controllers/FacetQueryController.ts

```diff
@@ -315,6 +315,9 @@ export class FacetQueryController {
     } else {
       mergeWith.constant = `${mergeWith.constant} ${this.additionalFilter}`;
     }
+    if (Utils.isEmptyString(mergeWith.basic)) {
```

Comment:
What does `mergeWith.basic` represent? I expected the condition to be `if not empty string` then we add the value as the constant query?

Reply:
I've re-phrased the code a bit, + your comment made me realize that there could be problems with constant expression.

In a nutshell, we're trying to "merge" with the currently being built facet query expression.

We want to make sure that if we have an additional filter, the basic expression should not be empty if we have one in the "standard" query, or we'll get skewed results if you also have another constant expression unknown to the UI (eg: set in the query pipeline).